draft-ietf-lake-authz.md

```diff
@@ -579,10 +579,14 @@ W verifies and decrypts ENC_U_INFO using the relevant algorithms of the selected
 
 In case OPAQUE_INFO is present, it is made available to the application.
 
-W calculates the hash of message_1 H(message_1), and associates this session identifier to the device identifier ID_U. If H(message_1) is not unique among session identifiers associated with this device identifier of U, the EDHOC session SHALL be aborted.
+W calculates the hash of message_1 H(message_1), and associates this session identifier to the device identifier ID_U. To avoid processing duplicates, H(message_1) SHALL be unique among session identifiers associated with a given ID_U.
+
```

Comment:
The original sentence is describe what action to take if message_1 has been received before. It is not clear from the new sentence how to make H(message_1) unique.

Reply:
Pushed an update that removes the text about "processing duplicates", since that's not the only possible reason for wanting unique session ids per device (this was discussed in a previous meeting).

#### Regarding your comment
> The original sentence is describe what action to take if message_1 has been received before

The answer is that the newly introduced phrase in line 585 ("If processing fails ...") serves as a catch all action for any kind of error happening in the above paragraphs.

> It is not clear from the new sentence how to make H(message_1) unique

I don't see how the previous version is more clear. I also think that just stating that it should be unique is enough.

Reply:
Pushed the solution proposed by @malishav. (now it doesn't use the SHALL in an incorrect way)